jconstraints-core/src/main/java/gov/nasa/jpf/constraints/expressions/ExpressionOperator.java

```diff
@@ -60,10 +60,6 @@ public static ExpressionOperator fromString(String str) {
     if (convertedOperator == null) {
       convertedOperator = FPComparator.fromString(str);
     }
-    if (convertedOperator == null) {
-      throw new UnsupportedOperationException(
-          "String " + str + " cannot be resolved to jConstraintsOperator");
-    }
```

Comment:
This seems to fail silently fail now – if the operator cannot be resolved, this now returns `null` instead of throwing.